src/main/java/fr/avenirsesr/portfolio/selfknowledge/application/adapter/dto/SelfKnowledgeCategoryDTO.java

```diff
@@ -0,0 +1,13 @@
+package fr.avenirsesr.portfolio.selfknowledge.application.adapter.dto;
+
+import fr.avenirsesr.portfolio.selfknowledge.domain.model.enums.ESelfKnowledgeCategoryType;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.UUID;
+
+@Schema(requiredProperties = {"id", "title", "description", "type"})
+public record SelfKnowledgeCategoryDTO(
+    UUID id,
+    String title,
+    String description,
+    @Schema(ref = "#/components/schemas/ESelfKnowledgeCategoryType")
+        ESelfKnowledgeCategoryType type) {}
```

Comment:
Je suis pas sûr que l'indentation soit nécessaire

```suggestion
    @Schema(ref = "#/components/schemas/ESelfKnowledgeCategoryType")
    ESelfKnowledgeCategoryType type) {}
```

Reply:
C'est google-java-format qui formatte les fichiers donc si je fais cette modif la CI va claquer je pense

Reply:
Ha c'est étonnant, sur mes PR c'était pas mis mais bon, c'est ok pour moi, j'approuve !
Merci @Nathan-Pignon !